config-model/src/test/java/com/yahoo/schema/derived/VsmFieldsTestCase.java

```diff
@@ -28,54 +28,52 @@ static Schema createSchema() {
         var sdoc = new SDDocumentType("test");
         schema.addDocument(sdoc);
         return schema;
-    }
-
-    private static VsmfieldsConfig vsmfieldsConfig(Schema schema) {
-        VsmFields vsmFields = new VsmFields(schema);
-        VsmfieldsConfig.Builder cfgBuilder = new VsmfieldsConfig.Builder();
-        vsmFields.getConfig(cfgBuilder);
-        return cfgBuilder.build();
-    }
+     }
+     private static VsmfieldsConfig vsmfieldsConfig(Schema schema) {
+         VsmFields vsmFields = new VsmFields(schema);
+         VsmfieldsConfig.Builder cfgBuilder = new VsmfieldsConfig.Builder();
+         vsmFields.getConfig(cfgBuilder);return cfgBuilder.build();
```

Comment:
It seems like the return statement was wrongly moved to the previous line?

Reply:
Falcon eye :)
Fixed in https://github.com/vespa-engine/vespa/pull/29897/commits/5ee662b82653ab871608fae7b80d10e0cf63b076